### PR TITLE
Adds ARG assault rifle to the game.

### PR DESCRIPTION
**Pull request by GoonOnMoon**
- Adds ARG assault rifle to the game. Used by NT fighting forces, but is able to be summoned by the wizard via summon guns.
- Removed the syndie tech origin, since it's a NT weapon.
- Added myself as a coder to admins.config

### CODE REVIEW

**Comment by GoonOnMoon** [comment]
http://imgur.com/c0onYox

Some screenshots.

**Comment by WJohn** [comment]
So it uses the same rounds as the syndicate carbine?

**Comment by GoonOnMoon** [comment]
5.56mm rounds, correct.

**Comment by WJohn** [comment]
Seems to be basically identical to it, can you at least swap from 3 round burst to single fire like the carbine? Is there any difference except for name and looks?

**Comment by GoonOnMoon** [comment]
ARG has no grenade launcher. Single fire would do nicely though, I'll add it in my next PR!

**Comment by WJohn** [comment]
You know you CAN make changes to your PR by making more commits and pushing that, right?

**Comment by Iamgoofball** [comment]
When can I do a ~tacticool reload~ by clicking on the gun with a new clip and it automatically kicking the old clip out and putting the new one in?

**Comment by GoonOnMoon** [comment]
@WJohn I do. I'm a bit busy with some other things and as I said, I'll change some stuff in my next PR, or if this isn't merged in a few days, a new commit.

@Iamgoofball is this already a feature?

File: config/admins.txt [hunk 1]
```diff
@@ -8,6 +8,7 @@
 # NOTE: syntax was changed to allow hyphenation of ranknames, since spaces are stripped.      #
 ###############################################################################################
 MrStonedOne = Host
+microscopics = Game Master
```
Comment (CorruptComputer) [inline config/admins.txt]: >putting yourself above everyone else

rude dude

**Comment by SconesC** [comment]
@GoonOnMoon Yes tactical reloads already are a thing

**Comment by Iamgoofball** [comment]
brb being tacticool

:+1:

**Comment by GoonOnMoon** [comment]
@Iamgoofball as in when you light a square with one and it's like "Withouy even breaking a stride..."

What do you guys think of me adding this to liberation station?

**Comment by palpatine213** [comment]
@GoonOnMoon Put yourself at the bottom of the text as is the proper ettiquite. Coders don't like a newcomer trying to show them up.

**Comment by GoonOnMoon** [comment]
@palpatine213 Noted.

**Comment by pudl** [comment]
@GoonOnMoon DON'T! keep it there! keep it there!

**Comment by Miauw62** [comment]
what is even the point of this.
traditionally, NT does not use ballistic weaponry, they use energy guns. Ballistic weapons beyond shotguns are generally reserved for syndies.
and if it's an admin gun, we already have plenty of OP guns admins can spawn.
no real reason to have this imo :-1:

**Comment by SconesC** [comment]
@Miauw62 I've actually wanted an NT bullet weapon for a while now though

**Comment by Miauw62** [comment]
alright, but why can't it be just a renamed child of one of the other bullet guns.

**Comment by GoonOnMoon** [comment]
@miauw62 It's in Liberation Station + summon guns.

**Comment by Dorsisdwarf** [comment]
There is an NT bullet weapon that's not a shotgun - the Mateba. Also, the saber SMG

**Comment by SconesC** [comment]
@Dorsisdwarf ...Is the saber in the code?

**Comment by GoonOnMoon** [comment]
@SconesC The Saber is the prototype SMG that can be made in R&D.

**Comment by Thunder12345** [comment]
Thank you again paprika for renaming all the guns

**Comment by Jordie0608** [comment]
You need to add the gun's path to the switch statement in rightandwrong, it doesn't get summoned just from being in the gunslist and will runtime without it.

Also changelog.

**Comment by GoonOnMoon** [comment]
Ah gotcha. Pretty new to git, how do I changelog @Jordie0608 ?

**Comment by phil235** [comment]
https://tgstation13.org/wiki/Guide_to_Changelogs

**Pull request closed without merging**